libs/execution/src/lib/blocks/block-execution-util.ts

```diff
@@ -31,18 +34,31 @@ export interface ExecutionOrderItem {
  */
 export async function executeBlocks(
   executionContext: ExecutionContext,
-  executionOrder: ExecutionOrderItem[],
+  compositeBlockTypeDefinition:
```

Comment:
This is not well named if we allow a `PipelineDefinition` as well. How about `pipelineContainer`, `container` or similar?

Reply:
Decided to go with `pipesContainer` in [021d04a](https://github.com/jvalue/jayvee/pull/491/commits/021d04a227e8ac9acc3f77fa14937e4ff8c037d9)